script/Arcueid/Day 4/04_00_ARC04_1.txt

```diff
@@ -287,7 +297,7 @@ The touch of the washcloth is enough to send a stinging pain through my throat.
 [sha:33e04eebd2591bbb4d3fef62fb2d955fb9930d89]{
 -- Page 24, Offset 5625.
 -- 「……なんだ、これ」
-"What is this...?"
+"What the hell...?"
```

Comment:
not more of them nooooooo
>What's... %{i}that%{/i}?

Reply:
It's a handy phrase! Suggestion kinda too strong / makes it sound like he's already spotted something when it comes before him actually looking in the mirror. Reads to me as more of a "why did my neck hurt" than a "what's that redness on my neck" comment.

Reply:
Maybe like

>That's... weird.

or something then